nautobot/cloud/__init__.py

```diff
@@ -0,0 +1 @@
+default_app_config = "nautobot.cloud.apps.CloudConfig"
```

Comment:
Just a pin that this will need to change under django 4 as `default_app_config` is replaced by `AppConfig.default = True`.